installer/uninstaller.sh

```diff
@@ -38,30 +38,60 @@ CHARTS_LIST="$RELEASE_PREFIX-mgt-api $RELEASE_PREFIX-crd $RELEASE_PREFIX-dex $RE
 HELM_LS_OUTPUT=`helm ls --output json`
 HELM_LIST=`jq --arg namearg "Releases" '.[$namearg]' <<< $HELM_LS_OUTPUT`
 K8S_NS_OUTPUT=`kubectl get ns --output=json`
-K8S_NS_LIST=`jq --arg namearg "items" '.[$namearg]' <<< $K8S_NS_OUTPUT`
+K8S_NAMESPACES=`echo $K8S_NS_OUTPUT | jq '.items[].metadata.name'`
 helm_arr=()
 K8S_NS_ARR=()
 
+[[ $K8S_NAMESPACES =~ "default-tenant" ]] && SINGLE_TENANT=1 || SINGLE_TENANT=0
+
 cd ../scripts
-./imm -k rm t default-tenant
-cd ../installer
+if [[ "$quiet" != "yes" ]]; then
+    echo "Uninstalling IMM. Following components will be removed:"
+    echo "*Helm releases (mgt-api, crd etc.)"
+    if [[ $SINGLE_TENANT == 1 ]]; then
+        echo "*Default tenant with it's resources(models,endpoints etc.)"
+    fi
+    echo "Are you sure you want to uninstall IMM? Y/n"
+    read DELETE_IMM
+else
+    DELETE_IMM="Y"
+fi
+if [[ $DELETE_IMM == "Y" ]]; then
 
-while read i; do
-   echo $i
-   RELEASE_NAME=`jq --arg namearg "Name" '.[$namearg]' <<< $i | tr -d '"'`
-   mark_release_to_be_deleted $RELEASE_NAME
-done <<<"$(jq -c '.[]' <<< $HELM_LIST)"
+    if [[ $SINGLE_TENANT == 1 ]]; then
+        echo "Deleting default-tenant..."
+        RESULT=`./imm -k rm t default-tenant`
+    else
+        RESULT=0
+    fi
 
+    if [[ "$RESULT" =~ "Tenant default-tenant deleted" || $RESULT == 0 ]]; then
+        cd ../installer
```

Comment:
So we only delete default tenant?

Reply:
In single tenant mode we delete default tenant and helm releases. In multi tenant mode we delete only helm releases as we don't know which namespaces belong to IMM tenants.

Reply:
I think it's an issue only when we have multiple IMM platform deployed at one cluster, am I right?

Reply:
As tenants have label which distinguishes on which platform they've been created, we can safely tell that `./imm ls t` will list only tenants created under that particular platform we want to uninstall. Please add removal of tenants which are listed by `ls t`

Reply:
One more thing: I think we can't delete only namespace - what with minio bucket? Minio could be not installed by us, in this case bucket from default tenant will be left